### DISCUSSION TITLE
Visual Studio C++ 2019

### DISCUSSION THREAD

**Discussion by chrixbedardcad** [google/oboe#1694] [Q&A]
Is this lib works with Visual Studio C++ 2019 ? My goal is to use this lib as a .dll for a VR project for Oculus Quest 2 using Unreal Engine 4.27.2. I try to have a very basic app build with VS C++ 2019 but I have an issue with the unistd.h.

Does anyone as a way to use that lib as a .dll for Visual Studio C++ ?

**Comment by philburk** [comment]
Oboe will **only** work on Android.
Are you cross compiling for Android using VS C++?

**Reply by chrixbedardcad** [reply]
Yes, I'm using VC C++ to build native content for Android through Unreal Engine.
